dojo/jira_link/helper.py

```diff
@@ -745,13 +749,12 @@ def add_jira_issue(obj, *args, **kwargs):
             priority_name=jira_priority(obj),
             epic_name_field=get_epic_name_field_name(jira_instance),
             duedate=duedate,
-            issuetype_fields=issuetype_fields)
+            issuetype_fields=issuetype_fields,
+            default_assignee=jira_project.default_assignee)
 
         logger.debug('sending fields to JIRA: %s', fields)
         new_issue = jira.create_issue(fields)
-        if jira_project.default_assignee:
-            jira.assign_issue(new_issue.key, jira_project.default_assignee)
-
```

Comment:
I see why you've done this here, but adding an additional check on the nee_issue to check if the assignee was set correctly would be even better. This way, there would have to be two failures that would need to occur for a jira issue to not have an assignee

Reply:
You mean to check for example if default_assignee is not empty? or check if the user exists?

Reply:
I mean to check if the jira issue has an assignee. If not, then the manual step of assigning a user can be used as a backup

Reply:
the proposed change only affects the use case when a ticket doesn't exist yet (add_jira_issue() function). The assignee is set for the new ticket at the moment of creation only when a 'default assignee' has been set in the Jira Project setting (at engagement or product level jira settings - depending on inheritance option). If no jira default assignee is set, then the current behaviour doesn't change and the ticket is still attempted to be created with no assignee.

Reply:
Apologies, I do not think I am doing a good job of making this clear. Here is a the flow I am trying to convey:
- User pushes ticket to Jira
- Default assignee is inserted into the options of the jira ticket creation function (the change you are introducing)
- After the ticket is created, check if the default assignee is indeed associated with the ticket (something new I am requesting)
- If the previous step is not true, then run the manual step of assigning the default assigned to the jira ticket (leaving the current behavior as a backup)
The extra step will hopefully not be called all that often, but is nice to have as a backup if the scenario were to arise

Please let me know If I can explain in another way

Reply:
got it now. Thank you for the clarification. I'll add the check

Reply:
This is exactly what I was looking for! Thank you :)